dev-tools/ecs-migration.yml

```diff
@@ -1209,6 +1209,14 @@
   comment: No alias mapping as field did not always exist
   beat: metricbeat
 
+## Modules
+
+### HTTP
+- from: http.request.body
+  to: http.request.body.content
+  alias: true
```

Comment:
Can't be aliased unfortunately, as the new field is moving directly under it. This means `http.request.body` can't be of type `alias`, since it has `.content` nested under it.

Reply:
Good point :-(